include/dgl/aten/macro.h

```diff
@@ -232,6 +232,14 @@
     });                                                     \
   });
 
+// Dispatching according to the context of array to enable CUDA UVA
+#define ATEN_CSR_SWITCH_CUDA_UVA(csr, array, XPU, IdType, op, ...) \
```

Comment:
Is it possible to check whether the graph is pinned in this macro?

Reply:
Can you explain it a bit more? Do you mean to put the `CHECK_VALID_CONTEXT` macro inside the  `ATEN_CSR_SWITCH_CUDA_UVA` macro to avoid confusion with `CHECK_SAME_CONTEXT` when adding a new op or something else?

Reply:
> Do you mean to put the `CHECK_VALID_CONTEXT` macro inside the `ATEN_CSR_SWITCH_CUDA_UVA` macro to avoid confusion with `CHECK_SAME_CONTEXT` when adding a new op

Yes.

Reply:
Resolved.